addons/website/views/snippets/snippets.xml

```diff
@@ -539,6 +539,12 @@
             <we-button data-customize-website="">Full</we-button>
             <we-button data-customize-website="website.option_layout_boxed_variables" id="option_layout_boxed">Boxed</we-button>
         </we-select>
+        <we-row string="Head" title="Enter code that will be added into the 'head' tag on every page of your side">
+            <we-button data-no-preview="true" data-open-custom-code-dialog="head" class="rounded"><t t-esc="'&lt;/&gt;'"/> Add</we-button>
```

Comment:
Why do you need a `t-esc` to print that ? Can't you just put the characters before the "Add" text ? Or maybe a font-awesome?